### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by ericpoirier**
Signed-off-by: Eric Poirier <eric.poirier@eclipse-foundation.org>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@Library('releng-pipeline') _
+
+hugo (
+  appName: 'eclipse.org/opensmartclide',
+  productionDomain: 'eclipse.org/opensmartclide',
```
Comment (ericpoirier) [inline Jenkinsfile:5]: I added this here because this is what I think is the production domain but we will have to confirm.

Comment (chrisguindon) [reply Jenkinsfile:5]: Project websites can't use eclispefdn-webdev project on OKD. This is not something they can use.

Comment (chrisguindon) [reply Jenkinsfile:5]: I made this PR to unblock @phkrief 

https://github.com/eclipse-opensmartclide/opensmartclide-website.src/pull/5/files

Comment (ericpoirier) [reply Jenkinsfile:5]: Sounds good, thanks.

**Pull request closed without merging**